staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -160,24 +194,24 @@ func newTestableController(
 	queueSetFactory fq.QueueSetFactory,
 ) *configController {
 	cfgCtlr := &configController{
-		queueSetFactory:        queueSetFactory,
-		obsPairGenerator:       obsPairGenerator,
-		serverConcurrencyLimit: serverConcurrencyLimit,
-		requestWaitLimit:       requestWaitLimit,
-		flowcontrolClient:      flowcontrolClient,
-		priorityLevelStates:    make(map[string]*priorityLevelState),
-	}
-	klog.V(2).Infof("NewTestableController with serverConcurrencyLimit=%d, requestWaitLimit=%s", serverConcurrencyLimit, requestWaitLimit)
-	cfgCtlr.initializeConfigController(informerFactory)
+		name:                       name,
+		clock:                      clock,
+		queueSetFactory:            queueSetFactory,
+		obsPairGenerator:           obsPairGenerator,
+		finishHandlingNotification: finishHandlingNotification,
+		asFieldManager:             asFieldManager,
+		foundToDangling:            foundToDangling,
+		serverConcurrencyLimit:     serverConcurrencyLimit,
+		requestWaitLimit:           requestWaitLimit,
+		flowcontrolClient:          flowcontrolClient,
+		priorityLevelStates:        make(map[string]*priorityLevelState),
+	}
+	klog.V(2).Infof("NewTestableController %q with serverConcurrencyLimit=%d, requestWaitLimit=%s", name, serverConcurrencyLimit, requestWaitLimit)
+	cfgCtlr.configQueue = workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(200*time.Millisecond, 8*time.Hour), "priority_and_fairness_config_queue")
+
 	// ensure the data structure reflects the mandatory config
 	cfgCtlr.lockAndDigestConfigObjects(nil, nil)
```

Comment:
Yes, because lockAndDigestConfigObjects ensures some minimal behavior even when there are no API objects that configure it.  The KEP describes how there is certain behavior that happens even without any configuration objects.